tests/integration_tests/data_steward/gcs_utils_test.py

```diff
@@ -2,6 +2,7 @@
 from io import open
 
 from googleapiclient.errors import HttpError
+from gcloud.gcs import StorageClient
 
 import gcs_utils
```

Comment:
Will the additional `gcs_utils` calls be refactored in a future ticket?  Or will this entire file go away when we have fully implemented google capabilities?

Reply:
So I have actually kinda struggled with how this might look, because things are getting pretty close to being finished.

I think this is the idea:

- in StorageClient, make get_hpo_bucket and get_drc_bucket return actual buckets and not strings.  gcs_utils continues to return strings.
- the rest of the code at this point still is using gcs_utils as well.
- with StorageClient now returning buckets, we can target files that already contain an instance of StorageClient as these are likely to be easier to transition.
- the rest of these files are then transitioned.
- gcs_utils loses get_*_bucket(...) when all files are using Bucket objects.
- audit gcs_utils, transfer any last functionality which will probably result in removing the file altogether.

Reply:
I think that could be a 13 story pointer easily